examples/pytorch/gat/README.md

```diff
@@ -2,54 +2,21 @@ Graph Attention Networks (GAT)
 ============
 
 - Paper link: [https://arxiv.org/abs/1710.10903](https://arxiv.org/abs/1710.10903)
-- Author's code repo (in Tensorflow):
+- Author's code repo (tensorflow implementation):
   [https://github.com/PetarV-/GAT](https://github.com/PetarV-/GAT).
 - Popular pytorch implementation:
   [https://github.com/Diego999/pyGAT](https://github.com/Diego999/pyGAT).
 
-Dependencies
-------------
-- torch v1.0: the autograd support for sparse mm is only available in v1.0.
-- requests
-- sklearn
-
-```bash
-pip install torch==1.0.0 requests
-```
-
 How to run
-----------
-
-Run with following:
-
-```bash
-python3 train.py --dataset=cora --gpu=0
-```
-
-```bash
-python3 train.py --dataset=citeseer --gpu=0 --early-stop
-```
-
-```bash
-python3 train.py --dataset=pubmed --gpu=0 --num-out-heads=8 --weight-decay=0.001 --early-stop
-```
+-------
 
+Run with the following (available dataset: "cora", "citeseer", "pubmed")
 ```bash
-python3 train_ppi.py --gpu=0
+python3 train.py --dataset cora
```

Comment:
Why did you drop the support of PPI?

Reply:
I was thinking of simplicity. 
Added PPI support back in case you think it is necessary.

Reply:
Also as a reminder here, I have removed "early stopping" function in both `train.py` and `train_ppi.py` for simplicity. Please let me know if you think if it is necessary and I will add it back.

Reply:
@jermainewang @yzh119 What do you think? Early stopping was added because some users complained about the accuracy.